internal/resolution/manifest/maven.go

```diff
@@ -136,10 +201,53 @@ func (m MavenManifestIO) Read(df lockfile.DepFile) (Manifest, error) {
 		EcosystemSpecific: MavenManifestSpecific{
 			Properties:                 properties,
 			RequirementsWithProperties: reqsWithProps,
+			RequirementsFromOtherPOMs:  otherRequirements,
 		},
 	}, nil
 }
 
+const MaxParent = 100
```

Comment:
is this a limit in Maven itself, or just for us?

Reply:
this is just for us

Reply:
Can you add a comment here clarifying that?

Reply:
done